src/components/content/UserMenu/index.tsx

```diff
@@ -36,6 +39,9 @@ export const UserMenu = ({
   userRole,
   children,
   top = 0,
+  width = 256,
+  position = 'absolute',
+  shadow = true,
```

Comment:
width and position are css styling attributes that should not be given directly as component props. Instead add an sx prop and write
`sx={{etc, top, ...sx}}`
to override any coded att value
Shadow as boolean okay from my pov

Reply:
additional style property will again leads `any` data type. Is that okay?